test/src/apidemos/chrome/testChromeAPI.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const common = require('../../../common.js');
+const nock = require('nock');
+const Nocks = require('../../../lib/nocks.js');
+const NightwatchClient = common.require('index.js');
+
+describe('chrome api demos', function () {
+  beforeEach(function () {
+    Nocks.cleanAll();
+    try {
+      Nocks.enable();
+    } catch (e) {}
+  });
+
+  afterEach(function () {
+    Nocks.deleteSession();
+  });
+
+  after(function () {
+    Nocks.disable();
+  });
+
+  it('run chrome api demo tests basic', function () {
+    const testsPath = path.join(__dirname, '../../../apidemos/chrome/chromeTest.js');
```

Comment:
A few remarks:
- why use `nock` here and Mockserver in the other one? Isn't it easier to use the same pattern? 
- there aren't any mocks for the chrome/firefox apis so I'm not sure what is going on here
- I think it would be easier to just write some basic protocol tests and add basic mocks for the chrome/firefox apis on the fakedriver
- we need to check all methods that are listed in the chrome/firefox api loaders to be available on the nightwatch api object

Reply:
Sure, I will update the tests 👍